example/spec/standalone/redeployment/TemplateOverrideDeploySpec.js

```diff
@@ -18,17 +18,17 @@ xdescribe('When an iam override template is in the IAM directory ', () => {
     try {
       await fs.mkdir('test_iam_override');
       await fs.copy('spec/standalone/redeployment/iam_override_template.yml', 'test_iam_override/cloudformation.template.yml');
-      let fileList = await fs.readdir('iam');
+      let fileList = await fs.readdir('app');
       fileList = fileList.filter((x) => x !== 'build');
-      const promiseList = fileList.map((filename) => fs.copy(`iam/${filename}`, `test_iam_override/${filename}`));
+      const promiseList = fileList.map((filename) => fs.copy(`app/${filename}`, `test_iam_override/${filename}`));
       await Promise.all(promiseList);
       await runKes(config, {
         kesCommand: 'compile',
         kesClass: 'node_modules/@cumulus/deployment/iam/kes.js',
         kesFolder: 'test_iam_override',
         template: 'node_modules/@cumulus/deployment/iam'
       });
-      cloudFormation = loadYmlFile('test_iam_override/cloudformation.yml');
+      cloudFormation = loadYmlFile('test_iam_override/iam.cloudformation.yml');
     } finally {
       await fs.remove('test_iam_override');
     }
```

Comment:
@L36, `  it('added a parameter', () => {` ->   it('has overridden the stack description', () => {`